src/test/java/edu/unc/lib/boxc/migration/cdm/services/RedirectMappingIndexServiceTest.java

```diff
@@ -140,8 +142,11 @@ public void redirectMappingIndexPopulatesTableCorrectly() throws Exception {
     }
 
     @Test
-    public void workIdsArePopulatedInRedirectMappingIndexForCompoundObjects() throws Exception {
-        List<String> nulls = new ArrayList<>();
+    public void tableIsPopulatedCorrectlyInRedirectMappingIndexForCompoundObjects() throws Exception {
+        List<String> cdm_object_ids = new ArrayList<>();
+        List<String> expected_ids = new ArrayList<>();
+        expected_ids.add("604");
+        expected_ids.add("607");
```

Comment:
you don't have to change this, but just so you know if you want to initialize a list with a few elements and don't need to change the list later, you can do this
```suggestion
        List<String> expected_ids = Arrays.asList("604", "607");
```

Reply:
Ugh i was looking for how to do that but didn't see anything. thanks for the tip!